public/index.html

```diff
@@ -27,7 +27,6 @@
     	<script src="/scripts/waitforimages.js"></script>
     	<script src="/scripts/gifshot.min.js"></script>
     	<script src="/scripts/gif.js"></script>
-    	<script src="/scripts/memento_toggle.js"></script>
 			<script src="/scripts/formatting.js?version=1005"></script>

```

Comment:
I think we do not need this version string here. Perhaps it was added to burst the cache for testing.

```suggestion
    	<script src="/scripts/formatting.js"></script>
```